.travis.yml

```diff
@@ -2,6 +2,6 @@ language: python
 install:
   - pip install coveralls
 script:
-  nosetests --with-coverage -v tex2Wiki DLMF_preprocessing maple2latex main_page eCF
+  nosetests --with-coverage -v DLMF_preprocessing maple2latex main_page eCF
```

Comment:
this removes the tex2Wiki from the coverage statistics. So this is some kind of cheeting.